### ISSUE TITLE
Slow loading for homepage and jobs list because of git repo jobs

### ISSUE THREAD

**Issue by FloLaco** [nautobot/nautobot#4852]
### Environment
* Nautobot version (Docker tag too if applicable):nautobot:1.5.16-py3.8
* Python version:3.8
* Database platform, version:postegre
* Middleware(s):

### Steps to Reproduce
1. Go to the main nautobot url (or click on the homepage)

### Expected Behavior
(very) fast loading and no log shown

### Observed Behavior
Some logs appears on nautobot frontend : 
```
09:15:21.490 INFO    nautobot.jobs :
  Repository successfully refreshed
INFO:nautobot.jobs:Repository successfully refreshed
09:15:22.589 INFO    nautobot.jobs :
  Repository successfully refreshed
INFO:nautobot.jobs:Repository successfully refreshed
09:15:23.542 INFO    nautobot.jobs :
  Repository successfully refreshed
INFO:nautobot.jobs:Repository successfully refreshed
09:15:24.562 INFO    nautobot.jobs :
  Repository successfully refreshed
INFO:nautobot.jobs:Repository successfully refreshed
09:15:25.487 INFO    nautobot.jobs :
  Repository successfully refreshed
INFO:nautobot.jobs:Repository successfully refreshed
09:15:26.510 INFO    nautobot.jobs :
  Repository successfully refreshed
INFO:nautobot.jobs:Repository successfully refreshed
09:15:27.520 INFO    nautobot.jobs :
  Repository successfully refreshed
INFO:nautobot.jobs:Repository successfully refreshed
09:15:28.492 INFO    nautobot.jobs :
  Repository successfully refreshed
INFO:nautobot.jobs:Repository successfully refreshed
09:15:29.758 INFO    nautobot.jobs :
  Repository successfully refreshed
INFO:nautobot.jobs:Repository successfully refreshed
09:15:30.793 INFO    nautobot.jobs :
  Repository successfully refreshed
INFO:nautobot.jobs:Repository successfully refreshed
```
The homepage appears only after the last log ! It seems that it's synchronous.

There is the same behaviour when I click on `Jobs` menu for listing all jobs but there are more logs (same log `Repository successfully refreshed`)

**Comment by lampwins** [comment]
This should mostly be a duplicate of #4142 I think, but good to call out the homepage bit. We are working through a solution now and it should cover both places.

**Comment by glennmatthews** [comment]
Duplicate of #3704? Fixed in v1.5.19 and later.

**Comment by lampwins** [comment]
We will close this one out and track the fix in #4142 in 1.6.
